### PR TITLE
Eselect EAPI bumps

### PR DESCRIPTION
**Pull request by dm0-**
@SoapGentoo Here are a couple trivial eselect module EAPI bumps for [EAPI5Removal](https://bugs.gentoo.org/698100).

### CODE REVIEW

**Review by junghans — APPROVED** [review]

**Comment by ulm** [comment]
> **app-eselect/eselect-ctags**: @gentoo/emacs
> **app-eselect/eselect-emacs**: @gentoo/emacs

I've cherry-picked the two packages maintained by @gentoo/emacs. Thanks.

File: app-eselect/eselect-notify-send/eselect-notify-send-0.1.ebuild [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
 HOMEPAGE="https://www.gentoo.org/proj/en/eselect/"
-SRC_URI=""
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~ia64 ~mips ppc ppc64 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
-IUSE=""
 
 RDEPEND=">=app-eselect/eselect-lib-bin-symlink-0.1.1
 	!<x11-libs/libnotify-0.7.5-r1"
```
Comment (ulm) [inline app-eselect/eselect-notify-send/eselect-notify-send-0.1.ebuild:14]: This should also have a run-time dependency on app-admin/eselect.

File: app-eselect/eselect-notify-send/eselect-notify-send-0.1.ebuild [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
-IUSE=""
 
-RDEPEND=">=app-eselect/eselect-lib-bin-symlink-0.1.1
+RDEPEND="app-admin/eselect
+	>=app-eselect/eselect-lib-bin-symlink-0.1.1
 	!<x11-libs/libnotify-0.7.5-r1"
-DEPEND=${RDEPEND}
 
-S=${FILESDIR}
+S=${WORKDIR}
```
Comment (telans) [inline app-eselect/eselect-notify-send/eselect-notify-send-0.1.ebuild:17]:
```suggestion
S="${WORKDIR}"
```

File: app-eselect/eselect-xvmc/eselect-xvmc-0.4.ebuild [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
-IUSE=""
 
-DEPEND=""
 RDEPEND="app-admin/eselect"
 
-S="${FILESDIR}"
+S=${WORKDIR}
```
Comment (telans) [inline app-eselect/eselect-xvmc/eselect-xvmc-0.4.ebuild:15]:
```suggestion
S="${WORKDIR}"
```

**Review by ulm — APPROVED** [review]

**Review by floppym — APPROVED** [review]
eselect-notify-send looks fine.

**Pull request closed without merging**